prime-router/src/main/kotlin/history/SubmissionHistory.kt

```diff
@@ -384,7 +398,7 @@ class DetailedSubmissionHistory(
              * the receivers.
              */
             return if (
-                reports.size > 1
+                reports.size > 1 && !hasNextAction
```

Comment:
Do you know why this is checking if `reports.size is > 1` instead of `reports.size is > 0`?

Reply:
I believe the assumption is that if we only have one report we must have only received it so far.